sdk/core/azure-core/CHANGELOG.md

```diff
@@ -3,10 +3,13 @@
 ## 1.5.0-beta.1 (Unreleased)
 
 ### Features Added
+
 - When a `RequestFailedException` exception is thrown, the `what()` method now includes information about the HTTP request which failed.
 
 ### Other Changes
 
```

Comment:
I believe that this counts as a breaking change and should be called out as such.